calcom/cal.com
# Fix issue that workflows sometimes need refresh to show updated data

## What does this PR do?

Fixes that sometimes the updated data was not shown correctly and a refresh was needed. 

Fixes #5274

Bug (before fix): https://www.loom.com/share/084c2bf87df344e2b9a6a15f5a28c52e

After fix: https://www.loom.com/share/b935bed6884b4e588c969f130f2541c1

**Environment**: Staging(main branch)

## Type of change


## How should this be tested?

- Test same as shown in Loom video 
- Test other versions of workflows and see if you see the updated data after saving
